recipes/intake-sql/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.2.0" %}
+{% set name = "intake-sql" %}
+{% set sha256 = "04e152dc5b5fdc428f08c348a559b303d452607bbfa2fdca645a7192f0328429" %}
+
+package:
+   name: {{ name }}
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }}  -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=3.5
```

Comment:
This constraint probably needs to be in both host and run.  It currently will be unconstrained at install time, and I don't think that's what you want.